[FIX] clarify definition of events in common principles (#595)
bids-standard/bids-specification: src/02-common-principles.md

    subject performed multiple conceptually different behaviors (with different
    sets of instructions) they will be considered one (combined) task.

> 1.  **Event** - something that happens or may be perceived as happening at a particular
>     moment during the recording.

Reviewer: Should we emphasize that "may be perceived as happening" refers to the perception of a test subject (it **does**, doesn't it??).

```suggestion
1.  **Event** - something that happens or may be perceived by a test subject as happening at a particular
    moment during the recording.
```

>     Events are most commonly associated with sudden-onset sensory stimulus presentations
>     or with the distinct onset of a subject motor action.

Reviewer: ```suggestion
    Events are most commonly associated with on- or offset of stimulus presentations,
    or with the distinct on- or offset of a subject's response or motor action.
```

>     Other events may include unplanned incidents,
>     changes in task instructions or experiment control parameters,
>     and noted data feature occurrences.

Reviewer: ```suggestion
    Other events may include unplanned incidents (e.g. sudden onset of noise and vibrations due to construction works, laboratory device malfunction etc.),
    changes in task instructions (e.g., switching the response hand) or experiment control parameters,
    and noted data feature occurrences.
```

Reviewer: I'm not sure what "changes in … experiment control parameters" is supposed to mean, do we have an example?

Same for "noted data feature occurrences" – you mean something like, "the experimenter realized one channel started to produce noise", this kind of occurrence?

Author: > something like, "the experimenter realized one channel started to produce noise", this kind of occurrence?

this is an example that seems to fall under the currently proposed definition, yes.

> I'm not sure what "changes in … experiment control parameters" is supposed to mean, do we have an example?

maybe the [eeg_matchingpennies](https://github.com/bids-standard/bids-examples/blob/master/eeg_matchingpennies/sub-05/eeg/sub-05_task-matchingpennies_events.tsv) dataset is an example for this ... we were doing a BCI game where the participant tried to outplay a computer reading the participant's EEG. over the course of the recording, the "experiment parameter" of BCI training changed (stage 1=bci classifier not trained, stage 2 and 3, classifier trained on data from previous stages) --> :man_shrugging:

Reviewer 2: A common example of "changes in experimental control parameters" comes from experiments organized into blocks during which the control variables are fixed for a series of trials. Suppose one of the control variables is the presentation rate, which is either "slow" or "fast". A given session might have two counter-balanced blocks , one block has the "slow" rate and the other the fast rate.  The starting and stopping points of each block are typically marked with events.  This information is extremely valuable for analysts.

Reviewer 2: I think that  "Events are most commonly associated with sudden-onset sensory stimulus presentations
    or with the distinct onset of a subject motor action."

would be better as:  "Events are most commonly associated with sudden-onset sensory stimulus presentations
    or with the distinct marker of a subject motor action."

Re:  Many experiments mark both onset and offset and other things.

Author: Thanks, I tried to address all of this in bb19602

    In BIDS, each event has an onset time and duration.

>     Note that not all tasks will have recorded events (e.g., "resting state").

Reviewer: ```suggestion
    Note that not all tasks will have recorded events (e.g., continuous recordings of "resting state" activity).
```

Author: I updated your suggestion to exclude "continuous" from "continuous recordings of resting state activity" ... because I think it's an unneeded constraint

    subject performed multiple conceptually different behaviors (with different
    sets of instructions) they will be considered one (combined) task.

1.  **Event** - something that happens or may be perceived by a test subject as happening
    at a particular instant during the recording.
    Events are most commonly associated with on- or offset of stimulus presentations,
    or with the distinct marker of on- or offset of a subject's response or motor action.
    Other events may include unplanned incidents
    (e.g., sudden onset of noise and vibrations due to construction work, laboratory device malfunction),
    changes in task instructions (e.g., switching the response hand),

>     or experiment control parameters (e.g., changing the stimulus presentation rate over experiment blocks),

Reviewer: ```suggestion
    or experiment control parameters (e.g., changing the stimulus presentation rate over experimental blocks),
```

    subject performed multiple conceptually different behaviors (with different
    sets of instructions) they will be considered one (combined) task.

1.  **Event** - something that happens or may be perceived by a test subject as happening
    at a particular instant during the recording.
    Events are most commonly associated with on- or offset of stimulus presentations,
    or with the distinct marker of on- or offset of a subject's response or motor action.
    Other events may include unplanned incidents
    (e.g., sudden onset of noise and vibrations due to construction work, laboratory device malfunction),
    changes in task instructions (e.g., switching the response hand),
    or experiment control parameters (e.g., changing the stimulus presentation rate over experimental blocks),

>     and noted data feature occurrences (e.g., a recording electrode producing noise).

Reviewer 3: `e.g.`s

Author: :man_facepalming: thanks
